modules/latex.php

```diff
@@ -17,8 +17,160 @@
  *
  * $latex e^{\i \pi} + 1 = 0$  ->  [latex]e^{\i \pi} + 1 = 0[/latex]
  * $latex [a, b]$              ->  [latex][a, b][/latex]
+ *
+ * In addition to supporting old markup, we also now support MathJax math delimiters:
```

Comment:
Yeah - currently we only support "inline" stuff.  I'm cool with adding support for display, but how would we communicate the difference to users? "For some stuff, do `$latex ...$`.  For other stuff, try `$$latex_display ...$$`." (or whatever).  Anyone who's worked with LaTeX before knows the difference, so maybe it's not a big deal.

As for backward compatibility, I'm totally OK with making things work better or adding capabilities :) We just can't make any of the existing stuff worse. (And we can't ruin existing content, like stuff that already includes the `\( ... \)` character sequence.)